recipes/transformers/LICENSE

```diff
@@ -0,0 +1,71 @@
+         Apache License
```

Comment:
It looks like this file is included in the tarball on PyPI so you don't need to include it.

(The `licence_file` attribute is still needed)

Reply:
i removed the LICENSE file

but in meta.yaml

should i 
```
licence_file: LICENSE
```
or
```
licence_file: ""
```
????

Reply:
It should be the filename of the file inside the tarball (which will almost certainly be the same as the name used on GitHub).

Reply:
thank, i think i modified it like you suggested.

Reply:
> It should be the filename of the file inside the tarball (which will almost certainly be the same as the name used on GitHub).

Sorry, I dont know where to ask, so I reply here.
These 2 packages haven't been uploaded yet.
https://github.com/conda-forge/sacremoses-feedstock
https://github.com/conda-forge/transformers-feedstock
Is there something wrong? 
How should I fix this?

Reply:
They’ve been affected by https://github.com/conda-forge/conda-smithy/issues/1154

It’s been fixed now for new recipes but you’ll need to push a trivial or empty commit to trigger those two.

Reply:
> They’ve been affected by [conda-forge/conda-smithy#1154](https://github.com/conda-forge/conda-smithy/issues/1154)
> 
> It’s been fixed now for new recipes but you’ll need to push a trivial or empty commit to trigger those two.

it worked!!! thank you!!!